### ISSUE TITLE
.all()

### ISSUE THREAD

**Issue by Miserlou** [Miserlou/NoDB#5]
This is another obvious omission.

**Comment by GavinPalmer1984** [comment]
- without caching
- assuming boto3 will not limit results
- ignoring memory concerns with previous assumption

```
    def all(self):
        """
        List all objects.
        """

        result = []

        bucket = self.s3.Bucket(self.bucket)
        for obj in bucket.objects.filter(Prefix=self.prefix).page_size(limit):
            serialized = obj.get()["Body"].read()
            deserialized = self._deserialize(serialized)
            result.append(deserialized['obj'])

        return result
```

**Comment by bendog** [comment]
fixed in #12
